02week/pigLatin.js

```diff
@@ -7,12 +7,37 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+//This checks to see if a vowel is present.
+const isAVowel = (letter) => {
+if(letter === 'a' || letter === 'e' || letter === 'i' || letter === 'o' || letter === 'u') {
```

Comment:
this could be simplified, just return the conditional

Reply:
Also, a better check would be an array of vowels, and then check to see if the letter is in the array.